templates/edge-computing/index.html

```diff
@@ -0,0 +1,425 @@
+        </li>
+        <li class="p-stepped-list__item u-no-padding--bottom">
+          <p class="p-stepped-list__title">Abstraction layer</p>
+        </li>
+      </ol>
+      <p>The edge of the network is the center of everyday businesses and applications.</p>
+      <p>Micro clouds empower ecosystems rather than single-vendor solutions, bringing apps closer to the consumers.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>What is the size of a micro cloud?</h2>
+      <p>Micro clouds go from tiny <a href="/kubernetes">Kubernetes</a> clusters in an outdoor box to small private edge clouds localised nearest to the consumers of an application. They can extend in site size, scale number of sites, and run with no on-site maintenance.</p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/23cbb344-Design.png",
+        alt="Micro clouds are meant to be customised to specific use cases",
+        width="244",
+        height="140",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru-accent is-dark">
+  <div class="u-fixed-width">
+    <div class="p-heading-icon--small">
+      <div class="p-heading-icon__header">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/4ab8ff35-Whitepaper+-+white.svg",
+          alt="",
+          width="171",
+          height="150",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+        <p class="p-heading--4">Whitepaper</p>
+      </div>
+      <h2 class="p-heading--4">CTOs' Guide to Micro Clouds</h2>
+      <p>This whitepaper demonstrates to Technical Leads the concept of micro clouds, a new class of compute for the edge and decentralised environments. Access the report today to get a condensed work of understanding edge computing constraints, values, graphics, and use cases.</p>
+      <p><a href="/engage/CTO-guide-to-micro-clouds-2021" class="p-button">Download the whitepaper</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-sv3">
+  <div class="u-fixed-width">
+    <p class="p-heading--3">Trade the exponential scalability of public and private clouds for security, privacy, governance, and low-latency of decentralised edge computing environments.</p>
+    <p>
+      <a href="/contact-us" class="p-button--positive js-invoke-modal">Connect with Canonical experts today</a>
+    </p>
+  </div>
+</section>
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/general" data-form-id="1257" data-lp-id="2154" data-return-url="https://ubuntu.com/contact-us/form/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
```

Comment:
The brief mentions:
> Unless updated otherwise, this first iteration will use the default contact form of the ubuntu.com website with a specific campaign for ‘Edge/MIcro cloud’

Do we need to get a new form id?

Reply:
@sowasred2012 I asked about this and was told to use the default one for now, but I will prompt again in case something have changed.